projects/css-custom-vars-viewer/src/CustomVarsViewer.ts

```diff
@@ -56,13 +56,13 @@ export class CustomVarsViewer extends SpectrumElement {
     static override styles = css`
         :host {
             display: block;
-            padding: var(--spectrum-global-dimension-size-300);
+            padding: var(--spectrum-spacing-200);
         }
         .picker-container {
-            padding-bottom: var(--spectrum-global-dimension-size-50);
+            padding-bottom: var(--spectrum-component-edge-to-visual-only-75);
```

Comment:
How do we know this is the desired variable?

Reply:
We don't! But among all other tokens that i could have used this was the closes one.

Reply:
Can we discuss about this with the tokens team first? There are many things which I would like to call out and the specifics needs to be clear before we make any update on the token variable names